run_site.sh

```diff
@@ -0,0 +1,5 @@
+npm install -g npm@latest
+npm install --save uswds@latest
```

Comment:
installing uswds here will override what is in the packages.json spec.  Recommend removing as this can cause unexpected changes in styles and relates us components

Reply:
Is there a different way to do it?  I just tested it like that, and without that command the `node_modules` directory does not build so it won't run.  I am only able to make it. work by including that command at startup.  ButI don't really know how npm works.  There is also something in here that looks like overrides for uswds, but I'm not sure if that's related: https://github.com/presidential-innovation-fellows/pif-website/tree/main/_assets/css/uswds-overrides

Reply:
Try simply `npm install` here instead of `npm install ... uswds` to install all `packages.json` spec dependencies, which will include `uswds`.

Reply:
Ok, I just tried replacing those two lines with only `npm install`, but the issue I am seeing now is that the background pictures in the blue banners and the LinkedIn favicon logo thing in the footer pages don't render.  Those images I believe are referenced in the hero and social-link files here: https://github.com/presidential-innovation-fellows/pif-website/tree/main/_assets/css/uswds-overrides. Is it an order of operations problem somewhere?

Reply:
Jekyll has some weird caching issues.  To resolve, I usually delete the `.jekyll_cache` and `_site` build folders, then rebuild fresh.

Reply:
Ahhh, yay!!! Thank you!  Now I know how to fix debugging my personal website, ha.  Looks good on my end now as far as I can tell with my latest commits.